db/nft.go

```diff
@@ -63,6 +64,57 @@ func GetClasses(conn *pgxpool.Conn, q QueryClassRequest, p PageRequest) (QueryCl
 	return res, nil
 }
 
+func GetClassesRanking(conn *pgxpool.Conn, q QueryRankingRequest) (QueryRankingResponse, error) {
+	sql := `
+	SELECT c.class_id, c.name, c.description, c.symbol, c.uri, c.uri_hash,
+	c.config, c.metadata, c.price,
+	c.parent_type, c.parent_iscn_id_prefix, c.parent_account, c.created_at, sold_count
+	FROM nft_class as c
+	JOIN (
+		SELECT c.id, count(n.id) as sold_count, array_remove(array_agg(DISTINCT n.owner), NULL) as owners
+		FROM iscn as i
+		JOIN nft_class as c ON i.iscn_id_prefix = c.parent_iscn_id_prefix
+		LEFT JOIN nft as n ON c.class_id = n.class_id
+			AND ($1 = true OR n.owner != i.owner)
+			AND ($2::text[] IS NULL OR n.owner != ALL($2))
+		WHERE ($4 = '' OR i.owner = $4)
+			AND ($5 = '' OR i.data #>> '{"contentMetadata", "@type"}' = $5)
+			AND ($6::jsonb IS NULL OR i.stakeholders @> $6)
+			AND ($7::jsonb IS NULL OR i.stakeholders @> $7)
```

Comment:
@nnkken Would this need to be changed after #35 is merged?

Reply:
The original `stakeholders` field is still kept. It should still work.

Reply:
The field was removed in latter commit in #35.

Reply:
Ok. I will update this after #35  merged.